helm_deploy/offender-search/templates/deployment.yaml

```diff
@@ -55,7 +57,7 @@ spec:
             failureThreshold: 10
           readinessProbe:
             httpGet:
-              path: /ping
+              path: /health
```

Comment:
I suspect this should be /health/ping i.e. allow this to be ready of this app is up. That would be consistent with other apps